python/tutorials/10-block-scaled-matmul.py

```diff
@@ -129,25 +129,43 @@ def block_scaled_matmul_kernel(  #
     elif output_type == 2:
         output_dtype = tl.float8e4nv
 
-    tl.inline_asm_elementwise("prefetch.tensormap [$1]; // dummy $0", "=r,l", [a_desc], dtype=tl.int32, is_pure=False,
-                              pack=1)
-    tl.inline_asm_elementwise("prefetch.tensormap [$1]; // dummy $0", "=r,l", [b_desc], dtype=tl.int32, is_pure=False,
-                              pack=1)
-    tl.inline_asm_elementwise("prefetch.tensormap [$1]; // dummy $0", "=r,l", [c_desc], dtype=tl.int32, is_pure=False,
-                              pack=1)
-
     pid = tl.program_id(axis=0)
     num_pid_m = tl.cdiv(M, BLOCK_M)
     pid_m = pid % num_pid_m
     pid_n = pid // num_pid_m
     offs_am = pid_m * BLOCK_M
-    offs_bn = pid_n * BLOCK_N
+    offs_bn_tma = pid_n * BLOCK_N
     offs_k = 0
+    offs_k_packed = 0
+    offs_bk = tl.arange(0, BLOCK_K // 2)
+    offs_bn = (pid_n * BLOCK_N + tl.arange(0, BLOCK_N)) % N
 
     ## block scale offsets
     offs_sm = (pid_m * (BLOCK_M // 128) + tl.arange(0, BLOCK_M // 128)) % M
     offs_sn = (pid_n * (BLOCK_N // 128) + tl.arange(0, BLOCK_N // 128)) % N
 
+    if MIXED_PREC_TMA:
+        b_desc = tl._experimental_make_tensor_descriptor(
+            b_desc_or_tensor,
+            shape=[N, K // 2],
+            strides=[K // 2, 1],
+            block_shape=[BLOCK_N, BLOCK_K // 2],
+        )
+        b_ptrs = None
```

Comment:
It seems I cannot prefetch tensormap here with device descriptor. Not sure if it's intended or if it's a bug

```
TypeError: cannot convert tensor_descriptor<<['256', '64'], uint8>> of type <class 'triton.language.core._experimental_tensor_descriptor'> to tensor
```